docs/python_support/quickstart.md

```diff
@@ -0,0 +1,199 @@
+# Python support in OVMS - quickstart {#ovms_docs_python_support_quickstart}
+
+OpenVINO Model Server allows users to write custom processing nodes in Python, so they may have full control over what happens with the data reaching such node and what comes out of it. Learn more about [Python support](). 
+
+In this quickstart you'll create a servable with a single custom Python code that will expect a string and return the same string, but all in uppercase. 
+
+To acheive that let's follow the steps:
+1. Prepare Docker Image And Workspace
+2. Prepare Python Code For The Server And Client
+3. Prepare Graph Configuration File
+4. Prepare Server Configuration File
+5. Run Model Server
+6. Send Request
+
+### Step 1: Prepare Docker Image And Workspace
+
+Since changing all the letters in the string to uppercase is a very basic use case, the basic Python-enabled model server image is enough. If you'd need some external modules (like opencv), you would need specifically add them. 
+In this quickstart it's not required, so just get the basic one:
+
+```
+docker pull openvino/model_server:latest
```

Comment:
This is not correct while the image should be built with special params
public image will have in the future different name probably with full tag